### DISCUSSION TITLE
Proposed change for the pi shield.

### DISCUSSION THREAD

**Discussion by MichaelTimmermans** [OpenScanEu/OpenScan-PCB#5] [Ideas]
I'd like to see the EN pins on the stepper shields be connected to one of the GPIO's on the RPI. That way you can configure the firmware to disable the steppers when the scanner is idle. 

This would waste less energy and the steppers do not sing all the time the scanner is powered up. They also have a chance to cool down between scans.

**Comment by probably-Erwins-Cat** [comment]
Is already included in a newer design (see discussion about HAT). 
The enable pin of the stepper are actually inverted, thus making it a disable pin.

**Reply by MichaelTimmermans** [reply]
Hey, yeah i saw that after making this post :)

**Comment by wicigee** [comment]
Any chance I can get the PCB printable layout for the schematic top and bottom? Can’t seem to find it. Might want to change to some locally available parts. Version 1 or 2(if working)
